events/wc_void_events.txt

```diff
@@ -852,152 +852,149 @@ narrative_event = {
 
 ### The Reawakening of C'thun ###
 # Ping event
-province_event = {
-	id = WCVOD.1000
+# province_event = {
+	# id = WCVOD.1000
```

Comment:
The event is commented but still tries to fire.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/88563581-32175e80-d043-11ea-9c5d-5f7a0701b209.png)
![image](https://user-images.githubusercontent.com/46384992/88563607-393e6c80-d043-11ea-980f-bf82b32a0718.png)

</details>

Reply:
Not fixed.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/88815354-5488b380-d1cc-11ea-8771-f01671177cb7.png)

</details>